### ISSUE TITLE
[Feature request] Take text note

### ISSUE THREAD

**Issue by IgorEliezer** [osmandapp/Osmand#7556]
Instead of creating a "Favorite" and adding a description to it, I would like that the addon "AV notes" were able to save written notes.

Sometimes it's not suitable to take a photo or record a voice note. Instead, a written note as a reminder or comment suits better.

**Comment by pebogufi** [comment]
The keypad in android is able to dictate text and to convert that to written words. Why you don't use that in favourites ?
I use it often, it works.

**Comment by vshcherb** [comment]
We have idea to make these buttons configurable

**Comment by ptrg** [comment]
I agree with this feature request. Handling text notes in the same fashion as voice notes would be welcome...

**Comment by martenjj** [comment]
Previously suggested as #5004.

**Comment by Noitarud** [comment]
I would like to also like to specify (or draw) the icon, and maybe also alert when in proximity (eg. There is an obstacle coming up, person's dog, conveniently missing speed limit sign), and maybe zoom level visibility (current as default).

**Comment by contrapunctus-1** [comment]
> For text notes (usually for OSM editing) I extensively use the GPX waypoint feature. You can add a waypoint at the current position or anywhere else on the map. You can add additional text, too. Afterwards when I'm home I export the GPX track and load it into JOSM for editing. Works great.

This is also my workflow. I use it in high-speed situations like surveying from a bus or cab (where taking photographs is impractical), but there are some issues with using OsmAnd for this -
1. You have to long-press on the map to add a waypoint which is not at your current location or a POI. This is slow and gets tiring over a long survey.
2. You have to select and delete the automatically-inserted waypoint text before you can write anything.

Other applications which can record tracks and waypoints don't have OsmAnd's killer combo of offline maps and hourly live updates, which is indispensable for surveying.

**Comment by phieti** [comment]
I'd prefer to simply have one more option for Media Notes plugin - Text, for me it just makes more sense
You press on widget, (like with audio or video) get popup text field, enter text, tap okay. Bam, its in your gpx track and everything else

So, in the end you can get gpx file with something liek this, but your text instead of photo names
<img width="1116" height="580" alt="Image" src="https://github.com/user-attachments/assets/da92e7ee-9cbb-4637-a148-335d8318d4b6" />

**Comment by scaidermern** [comment]
@phieti I use a quick exaction to do exactly this. Add a GPX waypoint with text.

**Comment by phieti** [comment]
> [@phieti](https://github.com/phieti) I use a quick exaction to do exactly this. Add a GPX waypoint with text.

Huh, interesting, will try.
